src/3p-frame.js

```diff
@@ -350,19 +350,20 @@ export const MessageType = {
 };
 
 /**
- * Serialize an AMP post message.
- *
- * @param type {string}
- * @param sentinel {string}
- * @param opt_data {Object=}
+ * Serialize an AMP post message. Output looks like:
+ * 'amp-011481323099490{"type":"position","sentinel":"12345","foo":"bar"}'
+ * @param {string} type
+ * @param {string} sentinel
+ * @param {Object=} opt_data
+ * @param {?string=} opt_rtvVersion
  * @returns {string}
  */
-export function serializeMessage(type, sentinel, opt_data) {
+export function serializeMessage(type, sentinel, opt_data, opt_rtvVersion) {
```

Comment:
Can user default values here (e.g. data = {}, rtvVersion = null).  Curious why not use $internalRuntimeVersion$?